<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddVisitWindow</class>
 <widget class="QDialog" name="AddVisitWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>674</width>
    <height>940</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dodaj wizytę</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>49</x>
     <y>29</y>
     <width>571</width>
     <height>841</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Data:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Lekarz przyjmujący:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Lekarz prowadzący</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Zabieg:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Opis:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Zalecenia:</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Następna wizyta:</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Użyte materiały:</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Użyte leki:</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Koszt wizyty:</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>Rodzaj płatności:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QDateEdit" name="visitDate">
      <property name="dateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2020</year>
        <month>1</month>
        <day>1</day>
       </datetime>
      </property>
      <property name="displayFormat">
       <string>yyyy-MM-dd</string>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="visitDoctor"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>Wywiad:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="visitDoctorMain"/>
    </item>
    <item row="3" column="1">
     <widget class="QTextEdit" name="visitTreatment"/>
    </item>
    <item row="4" column="1">
     <widget class="QTextEdit" name="visitInterview"/>
    </item>
    <item row="5" column="1">
     <widget class="QTextEdit" name="visitDescription"/>
    </item>
    <item row="6" column="1">
     <widget class="QTextEdit" name="visitRecommendation"/>
    </item>
    <item row="7" column="1">
     <widget class="QTextEdit" name="visitNextVisit"/>
    </item>
    <item row="8" column="1">
     <widget class="QTextEdit" name="visitTools"/>
    </item>
    <item row="9" column="1">
     <widget class="QTextEdit" name="visitMedicines"/>
    </item>
    <item row="10" column="1">
     <widget class="QLineEdit" name="visitCost"/>
    </item>
    <item row="11" column="1">
     <widget class="QLineEdit" name="visitPayment"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>880</y>
     <width>106</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Dodaj</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>880</y>
     <width>106</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Anuluj</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
